bundles/binding/org.openhab.binding.mios/examples/scripts/miosTransform.xslt

```diff
@@ -124,13 +125,16 @@ String   <xsl:value-of select="$DeviceNameFixed"/>DeviceStatus "<xsl:value-of se
 <xsl:when test="@service = 'urn:micasaverde-com:serviceId:AlarmPartition2'"      >AlarmPartition2</xsl:when>
 <xsl:when test="@service = 'urn:micasaverde-com:serviceId:Camera1'"              >Camera1</xsl:when>
 <xsl:when test="@service = 'urn:cd-jackson-com:serviceId:SystemMonitor'"         >SystemMonitor</xsl:when>
+<xsl:when test="@service = 'urn:cd-jackson-com:serviceId:Config'"                >CDJConfig</xsl:when>
```

Comment:
When writing plugins for MiOS/Vera, the plugin author has to come up with unique service names.  In this case our own Chris Jackson was the plugin author.  So it's not user-specific after all.